tests/legacy-cli/e2e/tests/packages/webpack/test-app.ts

```diff
@@ -30,6 +30,4 @@ export default async function (skipCleaning: () => void) {
     'DISABLE_V8_COMPILE_CACHE': '1',
   });
   await expectFileToMatch('dist/app.main.js', 'AppModule');
-
-  skipCleaning();
```

Comment:
This didn't seem to effect tests at all? This was the only use case of the "skip cleaning" and removing it didn't cause any tests to start failing.